src/Model/Media.php

```diff
@@ -14,6 +14,7 @@
 namespace Sonata\MediaBundle\Model;
 
 use Imagine\Image\Box;
+use Sonata\ClassificationBundle\Model\CategoryInterface;
```

Comment:
This line should be added for a reason, Can you check near line 600 where there is a string FQCN?

Reply:
Hmmm right, let me fix it.